core/src/main/java/org/testcontainers/containers/image/pull/policy/DefaultPullPolicy.java

```diff
@@ -0,0 +1,18 @@
+package org.testcontainers.containers.image.pull.policy;
+
+import java.util.Arrays;
+import lombok.extern.slf4j.Slf4j;
+import org.testcontainers.containers.image.ImageData;
+
+/**
+ * The default imagePullPolicy, which pulls the image from a remote repisotiry only if it does not exist locally
+ */
+@Slf4j
+public class DefaultPullPolicy implements ImagePullPolicy {
+
+    @Override
+    public boolean shouldPull(ImageData image) {
+        log.trace("Should pull image with tags: {}", Arrays.asList(image.getRepoTags()));
```

Comment:
not sure that this trace is needed, at least in this form

Reply:
Why not? I could remove it but that would make it inconsistent with the other policies, I think it's important for debugging purposes to understand which image pull policy is being used, no?